### PR TITLE
Removing Private Preview features from Swagger

### PR DESCRIPTION
**Pull request by mvvsubbu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by yungezz** [comment]
hi @mvvsubbu could you pls fix PrettierCheck by following guideline https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md#prettier-check? thanks.

**Comment by fengzhou-msft** [comment]
LGTM. please run PrettierCheck locally and fix it.

**Comment by mvvsubbu** [comment]
Fixed the PrettierCheck

File: specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json [hunk 1]
```diff
@@ -601,38 +597,6 @@
...
-          "description": "The SKU which should be used for this Storage Account"
-        },
-        "resourceTags": {
-          "$ref": "#/definitions/WorkspaceCustomObjectParameter",
-          "description": "A map of Tags which should be applied to the resources used by this Databricks Cluster."
-        },
-        "vnetAddressPrefix": {
-          "$ref": "#/definitions/WorkspaceCustomStringParameter",
-          "description": "The first 2 octets of the virtual network /16 address range (e.g., '10.139' for the address range 10.139.0.0/16)."
         }
```
Comment (ravrams) [inline specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json:600]: Why are we removing these?

Comment (mvvsubbu) [reply specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json:600]: Discussed with Premal and we don't have documentation or any usage examples, so wanted to remove it.

**Comment by fengzhou-msft** [comment]
@akning-ms @NullMDR can you help merge this PR? The removed peroperties are not released with SDKs before and they should be used for private preview.

**Pull request merged**